### ISSUE TITLE
Environment Variable Validation

### ISSUE THREAD

**Issue by kevinthedang** [kevinthedang/discord-ollama#104]
## Issue
* When running the application, the only environment variable that really shows issues when running the application is the `BOT_TOKEN`. All of them should provide some kind of error message/log that should display the issue in the console.

## Solution
* Implement some kind of environment variable checker to make sure that env variables provided are not the defaults from `.env.sample` and non-empty.

## References
* After #103 is merged, this should be `src/keys.ts`
* Referencing #101 for this issue
```ts
import { getEnvVar } from './utils/env.js'

export const Keys = {
    clientToken: getEnvVar('CLIENT_TOKEN'),
    model: getEnvVar('MODEL'),
    clientUid: getEnvVar('CLIENT_UID'),
    ipAddress: getEnvVar('OLLAMA_IP'),
    portAddress: getEnvVar('OLLAMA_PORT'),
} as const // readonly keys

export default Keys
```

**Comment by JT2M0L3Y** [comment]
@kevinthedang it appears that the `guild_id` environment variable was not fully removed from docker dependency. Opening another issue, #108.
